core/src/raw/adapters/typed_kv/api.rs

```diff
@@ -110,3 +117,47 @@ impl Value {
         size_of::<Metadata>() + self.value.len()
     }
 }
+
+/// Info for this key value accessor.
+pub struct Info {
+    scheme: Scheme,
+    name: String,
+    capabilities: Capability,
```

Comment:
We should define a `typed_kv::Capability` to simplify.

Reply:
And we can make all fields `pub` so that we don't need to add getter/setter functions for it.

Reply:
> We should define a `typed_kv::Capability` to simplify.

Because all typed_kv will share the same backend, so we only need know if adapter supports `get`/`set`/`delete`/`scan`.